src/main/java/com/catcher/batch/core/domain/entity/CatcherItem.java

```diff
@@ -46,4 +44,20 @@ public class CatcherItem extends BaseTimeEntity {
 
     @Column(name = "deleted_at")
     private ZonedDateTime deletedAt;
+
+    public void softDelete(){
+        this.deletedAt = ZonedDateTime.now();
```

Comment:
이 부분도 BaseTimeEntity 의 static필드를 protected로 가져가서 채우는건 어떤가용?
```suggestion
        this.deletedAt = ZonedDateTime.now(ZONE);
```